### ISSUE TITLE
Save button on book info page receives focus before entry fields

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1642]
### Description

The save button on the `Book Info` page receives focus _before_ the fields where users can enter book information. This is confusing for users of assistive technology and should be changed. Currently the page displays two `postbox-containers` and `save book information` is sequentially prior to `general book information`. The order should probably be reversed.

**Comment by connerbw** [comment]
We can't change the order, [WordPress sideboxes come before normal and advanced meta boxes.](https://github.com/WordPress/wordpress-develop/blob/9e3c57c2ffc428b53b0550ba55ae9597105c7899/src/wp-admin/edit-form-advanced.php#L635)

We can set an initial focus. Example: `	$('#pb_title').focus();`

Can go in: https://github.com/pressbooks/pressbooks/blob/dev/assets/src/scripts/book-information.js

**Comment by mcgratay** [comment]
Different behaviors on different supported browsers. 

Safari: wasn't able to focus on "Save" button at all
Chrome: "Save" button still receives focus before entry fields
Firefox: "Save" button still receives focus before entry fields

**Comment by connerbw** [comment]
@mcgratay and I tested together, it works as intended.

> We can't change the order, WordPress sideboxes come before normal and advanced meta boxes.
> We can set an initial focus. Example: $('#pb_title').focus();

We can't change HTML order, or use tabindex, because it's WordPress code.
